storybook/components/StoryWrapper/StoryWrapper.scss

```diff
@@ -13,6 +13,10 @@ $grid-gap: 1.5rem;
   }
 }
 
+.storyRowContainer + .storyRowContainer {
+  margin-top: 4rem;
+}
```

Comment:
You should be able to put this into the existing `.storyRowContainer` :)

```
.storyRowContainer {
  grid-area: stories;
  display: grid;
  gap: $grid-gap;

  & + .storyRowContainer { // I wonder if `& + &` works? Never tried
    margin-top: 4rem;
  }
}
```

Reply:
Yep I have updated to use & + & :)